src/foraging_gui/Foraging.py

```diff
@@ -1522,9 +1514,8 @@ def _load_most_recent_rig_json(self,error_if_none=True):
             rig_json_path = ''
             if error_if_none:
                 logging.error('Did not find any existing rig.json files')
-                self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata found!')
```

Comment:
![image](https://github.com/user-attachments/assets/dbc6c0d0-1885-4182-a341-5a169046c4de)

Reply:
This photo of MetadataWarning should sufficiently depict what MetadataWarning looks likes, even though different text is assigned to it.